src/commands/save-cache.yml

```diff
@@ -4,9 +4,13 @@ parameters:
     description: "The cache key to use. The key is immutable."
     type: string
     default: "go-mod"
+  path:
+    description: "Path to cache."
+    type: string
+    # /home/circleci/go is the GOPATH in the cimg/go Docker image
+    default: "~/go/pkg/mod"
```

Comment:
Please take a look at the git history for this file.

This was changed AWAY from using `~` or `GOPATH` because CircleCI doesn't parse either when saving the cache.

The fact that the default PATH only works on CircleCI is fine because on a non-CircleCI image, this PATH doesn't exist at all. The user would need to create one from scratch, in which case they'd need to set it here.

@gmemstr, @KyleTryon

Reply:
Reverted! Good call. I did test to make sure `~` would be expanded when saving the cache, but that might not cover some edge cases.